spec/helpers/blog_helper_spec.rb

```diff
@@ -3,6 +3,26 @@
 RSpec.describe BlogHelper do
   include BlogHelper
 
+  describe '#blog_posts_for_frontpage' do
+    subject(:posts) { blog_posts_for_frontpage }
+
+    before do
+      4.times { FactoryBot.create(:blog_post) }
```

Comment:
Should we make 5 to test that the spec below that's testing the limit isn't e.g. calling `.all`?

Reply:
The 5th object is created within the `it` block.

Reply:
🤦 doh